lib/rules/web/admin_console/4.4/operator_hub.xyaml

```diff
@@ -80,6 +80,59 @@ check_affected_pods_tab:
   params:
     tab_name: Affected Pods
   action: check_tab
+check_link_for_marketplace:
+  params:
+    text: Red Hat Marketplace
+    link_url: marketplace.redhat.com
+  action: check_link_and_text
+check_link_for_developer_catalog:
+  params:
+    text: Developer Catalog
+    link_url: catalog
+  action: check_link_and_text
+check_catalog_badge_by_checkbox:
+  action: click_checkbox_from_provider_type
+  elements:
+  - selector:
+      xpath: //*[contains(@class, 'pf-c-badge') and text()='<text>']
+    timeout: 30
+  action: click_checkbox_from_provider_type
+check_marketplace_operator_description_on_overlay:
+  params:
+    name: Marketplace
+    description: This Operator is purchased through Red Hat Marketplace
+  action: check_operator_description_on_overlay
+check_community_operator_description_on_overlay:
+  params:
+    name: Community
+    description: This is a community provided operator
+  action: check_operator_description_on_overlay
+check_operator_description_on_overlay:
+  action: click_checkbox_from_provider_type
```

Comment:
all sub actions should be moved to `check_community_operator_description_on_overlay` and `check_marketplace_operator_description_on_overlay `